src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -460,6 +460,7 @@ private void updateFoodPanel() {
 			isMeat.setEnabled(true);
 			isAlwaysEdible.setEnabled(true);
 			eatResultItem.setEnabled(true);
+			useDuration.setValue(32);

```

Comment:
It should only be changed if the user keeps it at 0. The user may wants a custom value.